tests/scripts/gen_release_notes.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+set -e
+
+function help() {
+  print="
+  suppose we want release note for v1.6.8, we'll check the list of pr merged in upstream/release-1.6 but not present in v1.6.7
+  EXAMPLE command:
+  ./tests/scripts/gen_release_notes.sh <rook/rook remote name>/release-1.6 v1.6.7
+  "
+  echo "$print"
+  exit 1
+}
+
+FROM_BRANCH=$1 # example branch name, "upstream/release-1.6"
+TO_BRANCH=$2 # example branch name, "v1.6.7"
+
+if [[ $# -ne 2 ]]; then
+  echo "requires exactly 2 arguments"
+  help
+fi
+
+GIT_REMOTE=$(git remote show 2>/dev/null |wc -l)
```

Comment:
Why not just `git remote show <remote name>`? There is no need to count the remote. I have 5 locally so this will fail for me.

Reply:
done